packages/teleport/src/services/clusters/makeCluster.ts

```diff
@@ -37,7 +37,7 @@ export default function makeCluster(json): Cluster {
     'proxyVersion',
   ]);
 
-  const connectedText = displayDateTime(lastConnected);
+  const connectedText = displayDateTime(new Date(lastConnected));
```

Comment:
```suggestion
  const lastConnectedDate = new Date(lastConnected)
  const connectedText = displayDateTime(lastConnectedDate);
```
then on line 44 of this file, remove the date constructor